internal/ingress/controller/config/config.go

```diff
@@ -559,6 +559,54 @@ type Configuration struct {
 	// Default: true
 	OpentracingTrustIncomingSpan bool `json:"opentracing-trust-incoming-span"`
 
+	// EnableOpentelemetry enables the nginx Opentelemetry extension
+	// By default this is disabled
+	EnableOpentelemetry bool `json:"enable-opentelemetry"`
+
+	// OpentelemetryConfig sets the opentelemetry config file
+	// Default: /etc/nginx/opentelemtry.toml
+	OpentelemetryConfig string `json:"opentelemetry-config"`
+
+	// OpentelemetryOperationName specifies a custom name for the server span
+	OpentelemetryOperationName string `json:"opentelemetry-operation-name"`
```

Comment:
Should it be "location span" instead of "server span" in the comment? Can `opentelemetry-operation-name` be set per location? It seems like something that should be specific to a location rather than being global to the server.

Reply:
This is similar to `opentracing-operation-name`. A name per ingress object, which can be overridden via annotation.